charms/build-and-release-easyrsa.sh

```diff
@@ -44,6 +44,5 @@ touch report.xml
 if [ ${RELEASE} = true ]; then
   CHARM=$(/usr/bin/charm push $JUJU_REPOSITORY/builds/easyrsa cs:~containers/easyrsa | head -n 1 | awk '{print $2}')
   echo "Releasing ${CHARM}"
-  ./charms/charm-release-with-latest-resources.sh ${CHARM} --channel ${RELEASE_TO_CHANNEL}
-  charm grant ${CHARM} everyone --channel ${RELEASE_TO_CHANNEL}
+  CHARM="$CHARM" FROM_CHANNEL=unpublished TO_CHANNEL=edge ./charms/promote-charm.sh
```

Comment:
Oh yeah, forgot about this. This removes a `RELEASE_TO_CHANNEL` param that we pass in from Jenkins.

I'm asserting that the automation for individual charms should only ever push to edge. We can promote from there separately.